.github/CODEOWNERS

```
@@ -44,6 +44,7 @@ go_deps.bzl               @dfinity/idx
 /publish/ @dfinity/idx
 
 # [Packages]
+/packages/icrc-cbor/                 @dfinity/finint
```

Is that something that should be co-owned?

```suggestion
/packages/icrc-cbor/                 @dfinity/finint @dfinity/cross-chain-team
```